tasks/run.js

```diff
@@ -39,3 +24,25 @@ async function runTask (name, args = []) {
     task.on('close', resolve)
   })
 }
+
+/**
+ * Creates a Gulp task for runTask()
+ *
+ * @param {string} name - npm script name
+ * @param {string[]} [args] - npm script arguments
+ * @returns {() => Promise<number>} Exit code
+ */
+const runTask = (name, args = []) => {
+  const script = `${name} ${args.join(' ')}`.trim()
```

Comment:
Maybe slightly nitpicky, but it seems that line is only used to generate the `displayName`. I got confused by why it was computed early, but then not passed to `task` down below, so I think it'd flow more clearly if it was right before the `runner.displayName` call.

Reply:
That's fine, can put it all on the same line

```mjs
// Add task alias
// https://gulpjs.com/docs/en/api/task#task-metadata
task.displayName = `npm run ${name} ${args.join(' ')}`.trim()
```